src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/RuntimePlatformCheckAnalyzerTests.cs

```diff
@@ -1,820 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow;
-using Xunit;
-using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
-    Microsoft.NetCore.Analyzers.InteropServices.RuntimePlatformCheckAnalyzer,
-    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
-
-namespace Microsoft.NetCore.Analyzers.InteropServices.UnitTests
-{
-    public class RuntimePlatformCheckAnalyzerTests
-    {
-        private readonly string PlatformCheckApiSource = @"
-namespace System.Runtime.InteropServices
-{
-    public class RuntimeInformationHelper
-    {
-        public static bool IsOSPlatformOrLater(OSPlatform osPlatform, int major) => true;
-        public static bool IsOSPlatformOrLater(OSPlatform osPlatform, int major, int minor) => true;
-        public static bool IsOSPlatformOrLater(OSPlatform osPlatform, int major, int minor, int build) => true;
-        public static bool IsOSPlatformOrLater(OSPlatform osPlatform, int major, int minor, int build, int revision) => true;
-
-        public static bool IsOSPlatformEarlierThan(OSPlatform osPlatform, int major) => true;
-        public static bool IsOSPlatformEarlierThan(OSPlatform osPlatform, int major, int minor) => true;
-        public static bool IsOSPlatformEarlierThan(OSPlatform osPlatform, int major, int minor, int build) => true;
-        public static bool IsOSPlatformEarlierThan(OSPlatform osPlatform, int major, int minor, int build, int revision) => true;
-    }   
-}";
-
-        [Fact]
-        public async Task SimpleIfTest()
```

Comment:
Where did all the flow analysis tests move to? It seems your new analyzer retained all the flow analysis logic from the prior analyzer, but not of the tests were migrated.

Reply:
Moved to the [PlatformCompatabilityAnalyzerTests.GuardedCalls.cs](src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzerTests.GuardedCalls.cs)

Reply:
Ah sorry. We follow the convention of all test files ending in `Tests.cs`, so I got confused. Maybe rename to `PlatformCompatabilityAnalyzerTests_GuardedCallTests.cs`?

Reply:
Sure